virt/virt-4-12-release-notes.adoc

```diff
@@ -134,20 +134,20 @@ The SVVP Certification applies to:
 //== Deprecated and removed features
 
 
-[id="virt-4-12-deprecated"]
-=== Deprecated features
-// NOTE: when uncommenting deprecated features list, change the header level below to ===
+// [id="virt-4-12-deprecated"]
+// === Deprecated features
+// // NOTE: when uncommenting deprecated features list, change the header level below to ===
 
-Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
-
-* In a future release, support for the legacy HPP custom resource, and the associated storage class, will be deprecated. Beginning in {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. The Operator continues to support the existing (legacy) format of the HPP custom resource and the associated storage class. If you use the HPP Operator, plan to xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[create a storage class for the CSI driver] as part of your migration strategy.
+// Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
 
 
 [id="virt-4-12-removed"]
 === Removed features
 
 Removed features are not supported in the current release.
 
+* Support for the legacy HPP custom resource, and the associated storage class, has been removed. Beginning in {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. 
+
```

Comment:
Beginning in {VirtProductName} {VirtVersion} - > Not sure whether you are stating when this change was made. If so it was made in OpenShift Virtualization 4.10. If you are referring to how thing works in the current version then change 'Beginning in'  to 'In'

Reply:
I feel like the "Only in the case of..." sentence is a bit wordy and awkward. 
Maybe change it to "Legacy HPP custom resource is supported only if it had been installed on a previous version of {VirtProductName}."

Reply:
lgtm